poradnia/letters/templates/letters/_letter_list.html

```diff
@@ -16,11 +16,11 @@
             </div>
             {% endif %}
             {% if object.html %}
-                <div class="embed-responsive embed-responsive-4by3">
-                    <iframe srcdoc="{{object.html|escape}}" sandbox="allow-top-navigation"></iframe>
-                </div>
+            <div class="embed-responsive embed-responsive-4by3">
+                <iframe srcdoc="{{object.html|escape}}" sandbox="allow-top-navigation"></iframe>
+            </div>
             {% else %}
-            <p>{{object.text|linebreaks}}</p>
+            <p>{{object.text|markdown}}</p>
```

Comment:
Może warto użyć bardziej generycznej nazwy np. dotyczącego standardowego formatowania treści?

Reply:
Świetnie.

Reply:
Czekam na aktualizacje PR.

Reply:
@ad-m zaktualizowałem PR. 
Jeśli wszystko wygląda OK to byłbym wdzięczny za dodanie labelki 'hacktoberfest-accepted' do PRka :)